cmds/upgraded/main.go

```diff
@@ -4,6 +4,8 @@ import (
 	"os"
 	"time"
 
+	"github.com/threefoldtech/zosv2/modules/zinit"
```

Comment:
I don't believe this entire file should be part of this Pull Request.

Reply:
yep indeed, I moved it to https://github.com/threefoldtech/zosv2/pull/155